articles/extensions/authorization-extension/v2/implementation/configuration.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Configuring the Authorization Extension
+description: How to configure the Authorization Extension
+toc: true
+---
+
+# Authorization Extension: Configuration
```

Comment:
I'm wondering, why is this step before the Setup step? I would expect first to configure my data at the extension and then to configure the "runtime parameters" (what to include in tokens, whether to persist data or not, etc)